low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -56,14 +57,46 @@ def validate(self):
             "cortx-sspl-test": None
             }
         # python 3rd party package dependency
-        pip3_3ps_packages_test = {
-            "Flask": "1.1.1"
+        pip3_packages_dep = {
```

Comment:
Rename `pip3_packages_dep` to `python_packages_dep` or `python3_packages_dep`

Reply:
okay @jjxsg 👍 , @cdeshmukh agree with `python3_package_dep`?

Reply:
yes, makes sense